src/main/kotlin/no/nav/familie/ef/iverksett/oppgave/OppgaveClient.kt

```diff
@@ -43,4 +46,40 @@ class OppgaveClient(
             )
         return response.data?.oppgaveId
     }
+
+    fun oppdaterOppgave(oppgave: Oppgave): Long {
+        val response = patchForEntity<Ressurs<OppgaveResponse>>(
+            URI.create("$oppgaveUrl/${oppgave.id!!}/oppdater"),
```

Comment:
Vet det ikke skal skje, men vil gjerne unngå `!!` i prod-kode. 
```suggestion
        val oppgaveId = oppgave.id ?: throw IllegalStateException("Oppgave mangler id")
        val response = patchForEntity<Ressurs<OppgaveResponse>>(
            URI.create("$oppgaveUrl/$oppgaveId/oppdater"),
```

Reply:
fixd [1174481](https://github.com/navikt/familie-ef-iverksett/pull/754/commits/1174481e18b968b2bf337bdba40113271ba5f68f)